tests/unit/swagger/pathGeneration/routeDuplicates.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import { MetadataGenerator } from '../../../../src/metadataGeneration/metadataGenerator';
+import { SpecGenerator2 } from '../../../../src/swagger/specGenerator2';
+import { getDefaultExtendedOptions } from '../../../fixtures/defaultOptions';
+
+it('should reject methods with same routes', () => {
+  expect(() => {
+    const invalidMetadata = new MetadataGenerator('./tests/fixtures/controllers/duplicateMethodsController.ts').Generate();
+    new SpecGenerator2(invalidMetadata, getDefaultExtendedOptions()).GetSpec();
```

Comment:
Do you need this line? Shouldn't it already throw above?

Reply:
You are right, will be fixed. Just bad copypaste .